silk/templates/silk/profiling.html

```diff
@@ -94,13 +94,13 @@ <h4>Profile</h4>
                    value="OverallTimeFilter"
                    name="filter-overalltime-typ"/>
             <input type="text"
-                   placeholder="seconds"
+                   placeholder="milliseconds"
```

Comment:
Is this a bugfix?

Reply:
I guess this is a typo fix, since the placeholder says 'seconds' but the filter actually accepts 'milliseconds', do you think it makes sense to make a separate PR for this?